Blackjack.py

```diff
@@ -0,0 +1,282 @@
+#This code is based on the Udemy course milestone project
+#This code will take the Class and Card classes to make a 52 card deck
+#Will ask the player to make a bet agains the dealer and then play blackjack
+#Player will be asek to hit or take another card
+#This is a bit unique as I forgot how to play Blackjack and so the program keeps playing
+#until all 52 cards are used up
+
+import random
+
+#These are my constants for the future I will UPPER_SNAKE_CASE
+suits = ('Hearts', 'Diamonds', 'Spades', 'Clubs')
+ranks = ('Two', 'Three', 'Four', 'Five', 'Six', 'Seven', 'Eight', 'Nine', 'Ten', 'Jack', 'Queen', 'King', 'Ace')
+values = {'Two':2, 'Three':3, 'Four':4, 'Five':5, 'Six':6, 'Seven':7, 'Eight':8, 'Nine':9, 'Ten':10, 'Jack':10,
+         'Queen':10, 'King':10, 'Ace':11}
+
+if __name__=="__main__":
+
+    class Card():
+
+        def __init__(self, suit, rank):
+            self.suit = suit
+            self.rank = rank
+            self.value = values[rank]
+
+        def __str__(self):

```

Comment:
Fyi, this is an example of using magic (dunder) methods to override functionality. 

Extra context: Alongside `__str__` there is also `__repr__`, they are very similary, but `__str__` is designed for end user output like you are doing here. And `__repr__` is more for provided debugging information, for repr you'd probably want to include the class name, `Card` and include `value` too. You don't need to do that here, I'm just providing extra info in case you see both usages, often together.

Reply:
Thanks I am adding this to the project so I can reference it in the future.